### PR TITLE
Modify comment syntax for code coverage

### PR DESCRIPTION
**Pull request by jatgarg**
## Description

Modify comment for code coverage.
Add the line coverage summary to the summary.

### CODE REVIEW

File: build-tools/packages/build-cli/src/codeCoverage/getCommentForCodeCoverage.ts [hunk 1]
```diff
@@ -71,7 +71,12 @@ const getCodeCoverageSummary = (
 	codeCoverageComparisonReport: CodeCoverageComparison[],
 ): string => {
 	const summary = codeCoverageComparisonReport
-		.sort((report1, report2) => report1.branchCoverageDiff - report2.branchCoverageDiff)
+		.sort(
+			(report1, report2) =>
+				report1.branchCoverageDiff +
+				report1.lineCoverageDiff -
+				(report2.branchCoverageDiff + report2.lineCoverageDiff),
+		)
```
Comment (tylerbutler) [inline build-tools/packages/build-cli/src/codeCoverage/getCommentForCodeCoverage.ts:79]:
```suggestion
		.sort(
			(report1, report2) =>
				(report1.branchCoverageDiff + report1.lineCoverageDiff) -
				(report2.branchCoverageDiff + report2.lineCoverageDiff),
		)
```

Is that what you intended? Either way, worth a comment explaining how this sorts the way you want and why.

Comment (jatgarg) [reply build-tools/packages/build-cli/src/codeCoverage/getCommentForCodeCoverage.ts:79]: Both mean the same but auto correct changes it to previous one even if I put it in paranthesis.

Comment (jatgarg) [reply build-tools/packages/build-cli/src/codeCoverage/getCommentForCodeCoverage.ts:79]: Not worthy of a comment imo. It is very much self explanatory imo.

File: build-tools/packages/build-cli/src/codeCoverage/getCommentForCodeCoverage.ts [hunk 2]
```diff
@@ -81,21 +86,23 @@ const getCodeCoverageSummary = (
...
 	}
 
-	if (codeCoverageBranchDiff > 0) {
-		return '<span style="color: green">⯅</span>';
+	if (codeCoverageDiff > 0) {
+		return "⯅";
 	}
 
-	return '<span style="color: red">⯆</span>';
+	return "<b>⯅</b>";
```
Comment (tylerbutler) [inline build-tools/packages/build-cli/src/codeCoverage/getCommentForCodeCoverage.ts:105]: What are the b tags for? If you're trying to draw attention to this color is probably a better way -- or maybe two glyphs -- `⯅⯅`

**Review by tylerbutler — COMMENTED** [review]
Looks good overall, but check the sort logic.

File: build-tools/packages/build-cli/src/codeCoverage/getCommentForCodeCoverage.ts (hunk 1 above)
Comment (tylerbutler) [reply build-tools/packages/build-cli/src/codeCoverage/getCommentForCodeCoverage.ts:79]: Well... I don't understand what it's doing and why, so can you share the obvious explanation?

**Review by tylerbutler — APPROVED** [review]

**Pull request merged**